stdlib/3/tokenize.pyi

```diff
@@ -40,7 +35,7 @@ class Untokenizer:
     def untokenize(self, iterable: Iterable[_Token]) -> str: ...
     def compat(self, token: Sequence[Union[int, str]], iterable: Iterable[_Token]) -> None: ...
 
-def untokenize(iterable: Iterable[_Token]) -> Any: ...
+def untokenize(iterable: Iterable[_Token]) -> bytes: ...
```

Comment:
I haven't tried but I think this returns a str if there is no ENCODING token in the input.

Reply:
You're right.  I looked again; the docstring is wrong.  Would you like this `Union[str, bytes]` instead of `Any`?

Reply:
`Any` is fine, we generally avoid Union return types.